### ISSUE TITLE
Facilitate ITS introduction and IP assignment increase/waiver

### ISSUE THREAD

**Issue by icflournoy** [RITlug/tasks#85]
Solomon has a contact at ITS that manages the club's IP allocation.

**Actions**
- Have eboard members (@Tjzabel ... ?) introduced to contact at ITS
- Learn what the process is for increasing or waivering the IP limit for the club, so that we can assign addresses to the CBET cluster

**Comment by Tjzabel** [comment]
This may have to wait for a bit, since Solomon is going on vacation for the full month of November.

**Comment by Serubin** [comment]
Yeah, apologies. I've been lagging on this. I'm going to put a date on this for first week of December.

@Tjzabel, @ct-martin, and @jrtechs are on my list for intros.

**Comment by Tjzabel** [comment]
@Serubin ping

**Comment by Tjzabel** [comment]
@Serubin I believe introductions have been made? Are there any follow-up tasks that now need to be done?
